forgotPass.php

```diff
@@ -0,0 +1,61 @@
+<?php
+require("model/DbConnection.php");
```

Comment:
fix indentation

Reply:
It's something gitHub or push issue , because its proper in my code.